rollup.config.js

```diff
@@ -0,0 +1,48 @@
+import babel from "@rollup/plugin-babel";
+import commonjs from "@rollup/plugin-commonjs";
+import json from "@rollup/plugin-json";
+import resolve from "@rollup/plugin-node-resolve";
+import replace from "@rollup/plugin-replace";
+import svgr from "@svgr/rollup";
+import analyze from "rollup-plugin-analyzer";
+import cleaner from "rollup-plugin-cleaner";
+import peerDepsExternal from "rollup-plugin-peer-deps-external";
+
+const plugins = [
+  cleaner({
+    targets: ["./dist/"],
+  }),
+  peerDepsExternal(),
+  replace({
+    "process.env.NODE_ENV": JSON.stringify("production"),
+    preventAssignment: true,
+  }),
+  resolve({
+    browser: true,
+    preferBuiltins: true,
+    extensions: [".js", ".jsx", ".svg"],
+  }),
+  svgr(),
```

Comment:
Why do you need svgr() plugin in this repo?

Reply:
It is not needed. I included it because its common in most of the packages rollup conflig(like [neeto-commons-frontend](https://github.com/bigbinary/neeto-commons-frontend/blob/main/rollup.config.js#L63) and [neeto-integrations-frontend](https://github.com/bigbinary/neeto-integrations-frontend/blob/main/rollup.config.js#L34)). Just to be on the safer side if in future we need to add SVGs.